pkg/virt-handler/seccomp/seccomp_test.go

```diff
@@ -0,0 +1,68 @@
+package seccomp
+
+import (
+	"os"
+	"path/filepath"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Secccomp", func() {
+
+	Context("Install", func() {
+
+		It("Should install", func() {
+			path, err := os.MkdirTemp("", "seccomp")
```

Comment:
Can we move the directory creation in a BeforeEach statement? I think we are also missing the directory clean-up.

Reply:
+1 Done